mimic/imimic.py

```diff
@@ -9,7 +9,8 @@
 
 class IAPIMock(Interface):
     """
-    An :obj:`IAPIMock` provides an API.
+    An :obj:`IAPIMock` provides an API via an entry in a tenant's service
+    catalog.
```

Comment:
Depending on how general the use case becomes it may make sense soonish to rename this such that it explicitly references Openstack Identity. Lots of APIs exist that don't have the same notion of a tenant or service catalog.